docs/developer-docs/functionality/ledger/deploy-new-token.md

```diff
@@ -101,11 +101,11 @@ This tutorial will guide you step-by-step to deploy your own token to the IC and
 
 Your new token is deployed and ready to be used.
 
-## Connect rosetta-api
+## Connect Rosetta
```

Comment:
Is it an official name change? it brings the question of the name of _our_ Rosetta vs the official naming:

> Rosetta is an open-source specification and set of tools that makes integrating with blockchains simpler, faster, and more reliable.

> The Rosetta APIs make a set of explicit design choices to enable simpler, faster, and more reliable blockchain integrations.

_https://www.rosetta-api.org/docs/welcome.html_

Reply:
Ah, I think I got it. Here you mean the specific rosetta-api application that was built for  the ledger canister? In this case it makes sense to revert the change.